app/code/Magento/SalesRule/Model/Rule.php

```diff
@@ -160,7 +160,7 @@ class Rule extends \Magento\Rule\Model\AbstractModel
     /**
      * @var \Magento\SalesRule\Model\Rule\Condition\Product\CombineFactory
      */
-    protected $_condProdCombineF;
+    protected $_condProdCombineFactory;
```

Comment:
Unfortunately, rename protected vars is backward incompatible according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/), please, revert